### PR TITLE
Updated variable.scss file for new UI

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#433 solved. @aka-jain Please review the PR.

### CODE REVIEW

**Comment by deshraj** [comment]
@aka-jain Can you review the PR?

File: frontend/src/css/modules/variables.scss [hunk 1]
```diff
@@ -40,6 +40,20 @@ $loader-dot-color: #ffaf4b;
...
+$gradient-blue: #323d8a;
+$gradient-purple: #af34d3;
+
+/*primary colors*/
+
+$primary-dark-blue: #382373;
+
+/*secondary colors*/
+
+$secondary-light-gray: #9d9d9d;
```
Comment (aka-jain) [inline frontend/src/css/modules/variables.scss]: remove primary and secondary form every variable please!

Comment (RishabhJain2018) [inline frontend/src/css/modules/variables.scss]: okk sure.

**Comment by RishabhJain2018** [comment]
@aka-jain Please review the PR.

**Comment by aka-jain** [comment]
@deshraj: merge this please (new ui).

**Pull request merged**